akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -299,13 +317,12 @@ final class ORMap[A, B <: ReplicatedData] private[akka] (
    * Note that if there is a conflicting update on another node the entry will
    * not be removed after merge.
    */
-  def -(key: A)(implicit node: Cluster): ORMap[A, B] = remove(node, key)
+  def remove(key: A)(implicit node: SelfUniqueAddress): ORMap[A, B] = remove(node.uniqueAddress, key)
```

Comment:
For Java API I think this method can be used `remove(key, selfUniqueAddress)`, but since the other methods have the `node` as the first parameter I think we should also add an overload:
```
def remove(node: SelfUniqueAddress, key: A): ORMap[A, B]
```